html/pfappserver/root/static.alt/src/globals/pfFormatters.js

```diff
@@ -41,5 +41,20 @@ export const pfFormatters = {
       default:
         return null
     }
+  },
+  genderFromString: (value, key, item) => {
+    if (value === null || value === '') return null
+    switch (value.toLowerCase()) {
+      case 'm':
+      case 'male':
+      case 'man':
+        return 'm'
+      case 'f':
+      case 'female':
+      case 'woman':
+        return 'f'
+      default:
+        return null
```

Comment:
What about LGBT?

Reply:
Which genders should we support?

 - [ ] Agender
 - [ ] Androgyne
 - [ ] Androgynous
 - [ ] Bigender
 - [ ] Cis
 - [ ] Cisgender
 - [ ] Cis Female
 - [ ] Cis Male
 - [ ] Cis Man
 - [ ] Cis Woman
 - [ ] Cisgender Female
 - [ ] Cisgender Male
 - [ ] Cisgender Man
 - [ ] Cisgender Woman
 - [ ] Female to Male
 - [ ] FTM
 - [ ] Gender Fluid
 - [ ] Gender Nonconforming
 - [ ] Gender Questioning
 - [ ] Gender Variant
 - [ ] Genderqueer
 - [ ] Intersex
 - [ ] Male to Female
 - [ ] MTF
 - [ ] Neither
 - [ ] Neutrois
 - [ ] Non-binary
 - [ ] Other
 - [ ] Pangender
 - [ ] Trans
 - [ ] Trans*
 - [ ] Trans Female
 - [ ] Trans* Female
 - [ ] Trans Male
 - [ ] Trans* Male
 - [ ] Trans Man
 - [ ] Trans* Man
 - [ ] Trans Person
 - [ ] Trans* Person
 - [ ] Trans Woman
 - [ ] Trans* Woman
 - [ ] Transfeminine
 - [ ] Transgender
 - [ ] Transgender Female
 - [ ] Transgender Male
 - [ ] Transgender Man
 - [ ] Transgender Person
 - [ ] Transgender Woman
 - [ ] Transmasculine
 - [ ] Transsexual
 - [ ] Transsexual Female
 - [ ] Transsexual Male
 - [ ] Transsexual Man
 - [ ] Transsexual Person
 - [ ] Transsexual Woman
 - [ ] Two-Spirit

Reply:
Adding `o`/`other` for 3rd gender selection

https://en.wikipedia.org/wiki/Third_gender